packages/batcher/webserver/src/index.ts

```diff
@@ -419,6 +424,10 @@ async function initializeServer(
             );
             setWebserverClosed();
           } catch (err) {
+            if ((err as Error).message === 'Recaptcha validation failed') {
```

Comment:
Probably it's better to define a custom error type and use `instanceof` instead of this